src/Sarif/Core/FileData.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.
```

Comment:
Remind me why this file exists? We auto-generate `FileData` (assuming you've got the correct code gen hint, but I put that in earlier today).